frameworks/keyed/vanillajs-2/src/Main.js

```diff
@@ -0,0 +1,205 @@
+'use strict';
+
+function _random(max) {
+    return Math.round(Math.random()*1000)%max;
+}
+
+const rowTemplate = document.createElement("tr");
```

Comment:
You can try adding a space in the actual template for where the dynamic text nodes will appear. This will create a text node with just a space, which you can then update using `nodeValue` rather than using `textContent`